app/services/puppet_class_importer.rb

```diff
@@ -142,7 +142,7 @@ def old_environments
   def db_classes(environment)
     return @foreman_classes[environment] if @foreman_classes[environment]
     return [] unless (env = Environment.find_by_name(environment))
-    @foreman_classes[environment] = env.puppetclasses.includes(:lookup_keys, :class_params)
+    @foreman_classes[environment] = env.puppetclasses
```

Comment:
Looks like this isn't being used anywhere (bullet is complaining and I couldn't find anywhere it is used).